ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -81,6 +93,45 @@ public static StatementExecutorResponse execute(
         .map(err -> new ErrorEntity(statement.getStatementText(), err)));
   }
 
+  private static List<String> validate(final CreateConnector createConnector,
+      final ConnectClient client) {
+    final Map<String, String> config = new HashMap<>(createConnector.getConfig().size());
+    createConnector.getConfig().forEach((k, v) ->
+        // Parsing the statement wraps string fields with single quotes which breaks the
+        // validation.
+        config.put(k, v.toString().replace("'", "")));
```

Comment:
Is it safe to simply replace all single-quotes like this? It might be safer to instead use something similar to https://github.com/confluentinc/ksql/blob/e4c229517173d1f3feac6b29c1b1f88f9f4f8a43/ksqldb-parser/src/main/java/io/confluent/ksql/util/ParserUtil.java#L98

Reply:
Ah cool, I didn't know about this one. Thanks for pointing it out!